packages/compiler/src/config/config-loader.ts

```diff
@@ -21,7 +21,8 @@ export const defaultConfig = deepFreeze({
  */
```

Comment:
The doc for this function seems a little inaccurate with this new behavior, should maybe be updated or split in 2 functions

Reply:
I was able to revert these changes while still making all old and new tests pass.